diff --git a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/FusedLocationSource.java b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/FusedLocationSource.java
index ca65bd7..cb47fe4 100644
--- a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/FusedLocationSource.java
+++ b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/FusedLocationSource.java
@@ -10,7 +10,6 @@ import com.google.android.gms.location.LocationCallback;
 import com.google.android.gms.location.LocationRequest;
 import com.google.android.gms.location.LocationResult;
 import com.google.android.gms.location.LocationServices;
-import com.google.android.gms.location.Priority;
 import com.google.android.gms.maps.LocationSource;
 import com.google.android.gms.tasks.OnSuccessListener;
 
@@ -26,7 +25,7 @@ public class FusedLocationSource implements LocationSource {
         fusedLocationClientProviderClient =
                 LocationServices.getFusedLocationProviderClient(context);
         locationRequest = LocationRequest.create();
-        locationRequest.setPriority(Priority.PRIORITY_HIGH_ACCURACY);
+        locationRequest.setPriority(LocationRequest.PRIORITY_HIGH_ACCURACY);
         locationRequest.setInterval(5000);
     }
 
diff --git a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapManager.java b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapManager.java
index fb78287..c371c3c 100644
--- a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapManager.java
+++ b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapManager.java
@@ -20,7 +20,7 @@ import com.facebook.react.uimanager.ThemedReactContext;
 import com.facebook.react.uimanager.ViewGroupManager;
 import com.facebook.react.uimanager.annotations.ReactProp;
 import com.facebook.react.uimanager.events.RCTEventEmitter;
-import com.google.android.gms.location.Priority;
+import com.google.android.gms.location.LocationRequest;
 import com.google.android.gms.maps.GoogleMap;
 import com.google.android.gms.maps.GoogleMapOptions;
 import com.google.android.gms.maps.MapsInitializer;
@@ -43,10 +43,10 @@ public class MapManager extends ViewGroupManager<MapView> {
     );
 
     public static final Map<String, Integer> MY_LOCATION_PRIORITY = MapBuilder.of(
-            "balanced", Priority.PRIORITY_BALANCED_POWER_ACCURACY,
-            "high", Priority.PRIORITY_HIGH_ACCURACY,
-            "low", Priority.PRIORITY_LOW_POWER,
-            "passive", Priority.PRIORITY_PASSIVE
+            "balanced", LocationRequest.PRIORITY_BALANCED_POWER_ACCURACY,
+            "high", LocationRequest.PRIORITY_HIGH_ACCURACY,
+            "low", LocationRequest.PRIORITY_LOW_POWER,
+            "passive", LocationRequest.PRIORITY_NO_POWER
     );
 
     private final ReactApplicationContext appContext;
